docs/book/how-to/develop-locally/keep-your-dashboard-server-clean.md

```diff
@@ -0,0 +1,160 @@
+    
+    # Get the list of pipeline runs created in the last 24 hours
+    recent_runs = zc.list_pipeline_runs(created=f"gt:{time_filter}")
+    
+    # Delete each run
+    for run in recent_runs:
+        print(f"Deleting run: {run.id} (Created: {run.body.created})")
+        zc.delete_pipeline_run(run.id)
+    
+    print(f"Deleted {len(recent_runs)} pipeline runs.")
+
+if __name__ == "__main__":
+    delete_recent_pipeline_runs()
+```
+
+For different time ranges you can update this as appropriate.
+
+## Pipelines
+
+### Deleting Pipelines
+
+Pipelines that are no longer needed can be deleted using the command:
+
+```bash
+zenml pipeline delete <PIPELINE_ID_OR_NAME>
+```
+
+This allows you to start fresh with a new pipeline, removing all previous runs
+associated with the deleted pipeline. This is a slightly more drastic approach,
+but it can sometimes be useful to keep the development environment clean.
+
+## Unique Pipeline Names
+
+Pipelines can be given unique names each time they are run to uniquely identify them. This helps differentiate between multiple iterations of the same pipeline during development.
+
+By default ZenML generates names automatically based on the current date and
+time, but you can pass in a `run_name` when defining the pipeline:
+
+```python
+training_pipeline = training_pipeline.with_options(
+    run_name="custom_pipeline_run_name"
+)
+training_pipeline()
+```
+
+Note that pipeline names must be unique. For more information on this feature,
+see the [documentation on naming pipeline runs](../build-pipelines/name-your-pipeline-and-runs.md).
+
+## Models
+
+Models are something that you have to explicitly register or pass in as you
+define your pipeline, so to run a pipeline without it being attached to a model
+is fairly straightforward: simply don't do the things specified in our
+[documentation on registering
+models](../use-the-model-control-plane/register-a-model.md).
+
+In order to delete a model or a specific model version, you can use the CLI or
+Python SDK to accomplish this. As an example, to delete all versions of a model,
+you can use:
+
+```bash
+zenml model delete <MODEL_NAME>
+```
+
+See the full documentation on [how to delete models](../use-the-model-control-plane/delete-a-model.md).
+
+## Artifacts
+
+### Pruning artifacts
+
+If you want to delete artifacts that are no longer referenced by any pipeline
+runs, you can use the following CLI command:
+
+```bash
+zenml artifact prune
+```
+
+By default, this method deletes artifacts physically from the underlying artifact store AND also the entry in the database. You can control this behavior by using the `--only-artifact` and `--only-metadata` flags.
+
+For more information, see the [documentation for this artifact pruning feature](../handle-data-artifacts/delete-an-artifact.md).
+
+## Cleaning your environment
+
+As a more drastic measure, the `zenml clean` command can be used to start from
```

Comment:
im not sure if zenml clean works reliably in a deployed zenml scenario so maybe mention that?